common/unsafe/src/main/java/org/apache/spark/sql/catalyst/util/CollationAwareUTF8String.java

```diff
@@ -1166,6 +1169,60 @@ public static UTF8String trimRight(
     return UTF8String.fromString(src.substring(0, charIndex));
   }
 
+  public static UTF8String[] splitSQL(final UTF8String input, final UTF8String delim,
```

Comment:
why do we call it `splitSQL` instead of just `split`?

Reply:
by Spark terminology, `split` refers to splitting on a regex, while `splitSQL` refers to splitting on a delimiter (in this expression, we want the latter)